setup.py

```diff
@@ -43,7 +43,7 @@
     ],
     packages=[
         "sdc.crypto",
-        "sdc.crypto.utils",
```

Comment:
Can we package the script up so it's available from the command line after installing the package? 
http://python-packaging.readthedocs.io/en/latest/command-line-scripts.html